src/main/java/teammates/ui/webapi/output/StudentProfileData.java

```diff
@@ -0,0 +1,62 @@
+package teammates.ui.webapi.output;
+
+import teammates.common.datatransfer.attributes.StudentProfileAttributes;
+import teammates.common.datatransfer.attributes.StudentProfileAttributes.Gender;
+
+/**
+ * The API output for the student profile.
+ */
+public class StudentProfileData extends ApiOutput {
+
+    private String name;
+    private String shortName;
+    private String email;
+    private String institute;
+    private String nationality;
+    private Gender gender;
+    private String moreInfo;
+    private String pictureKey;
```

Comment:
I think either you or @rrtheonlyone can remove the picture key, depending on who's PR is merged first.

Reply:
However the existing frontend code utilizes the `pictureKey` attribute